### PR TITLE
Fixed Verifier problem for constraints with the same type

### PR DESCRIPTION
**Pull request by konefalg**

### CODE REVIEW

File: whylog/constraints/constraint_manager.py [hunk 1]
```diff
@@ -37,15 +37,20 @@ def get_types(cls):
 
 class ConstraintManager(object):
     """
-    there should be one such object per rule being verified
+    There should be one such object per rule being verified.
+    ConstraintManager collects constraints objects, one per each constraint.
```
Comment (kgromadzki) [inline whylog/constraints/constraint_manager.py]: One constraint object per constraint type?

File: whylog/constraints/verifier.py [hunk 2]
```diff
@@ -172,7 +172,7 @@ def single_constraint_not(cls, clues_lists, effect, constraint, constraint_manag
         """
         clues_lists = cls._construct_proper_clues_lists(clues_lists)
         for combination in cls._clues_combinations(clues_lists):
-            if cls._verify_constraint(combination, effect, constraint, constraint_manager):
+            if cls._verify_constraint(combination, effect, 0, constraint, constraint_manager):
```
Comment (kgromadzki) [inline whylog/constraints/verifier.py:175]: Add some explanation that this 0 idx came form assumption that has just one constraint. Also some const for that.

File: whylog/constraints/constraint_manager.py [hunk 3]
```diff
@@ -37,15 +37,20 @@ def get_types(cls):
...
+    so constraints in _actual_constraints has the same numeration
+    as in corresponding rule.
     """
 
     def __init__(self):
         self._actual_constraints = {}
 
-    def __getitem__(self, constraint_data):
-        constraint = self._actual_constraints.get(constraint_data['name'])
+    def get_constraint_object(self, idx, constraint_data):
```
Comment (kgromadzki) [inline whylog/constraints/constraint_manager.py]: Rename idx. Idx isn't a good name

File: whylog/constraints/verifier.py (hunk 2 above)
Comment (konefalg) [inline whylog/constraints/verifier.py:175]: I've added a comment.

**Pull request merged**